src/python/pants/core/register.py

```diff
@@ -91,15 +93,17 @@ def rules():
 def target_types():
     return [
         ArchiveTarget,
-        FileTarget,
+        DockerEnvironmentTarget,
         FilesGeneratorTarget,
+        FileTarget,
         GenericTarget,
-        ResourceTarget,
-        ResourcesGeneratorTarget,
-        RelocatedFiles,
         LocalEnvironmentTarget,
-        DockerEnvironmentTarget,
+        LockfilesGeneratorTarget,
+        LockfileTarget,
```

Comment:
Sorry for the noise, sorted this list alphabetically.

The two `Lockfiletarget` and `LockfilesGeneratorTarget` entries are the only real changes here.